### PR TITLE
Wrote intro to npi vignette

### PR DESCRIPTION
**Pull request by parmsam**
Wrote introduction to npi package vignette for #13

### CODE REVIEW

File: vignettes/npi.Rmd [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+Other search arguments for the function include npi number, enumeration type, taxonomy description, individual provider's first name, individual provider's last name, use first name alias, organization name, address purpose, state, postal code, country code, and limit.
+
+Visit the function's help page via `?npi_search` after installing and loading the package for more details.
+
+## Increasing number of records returned
+
+The `limit` argument lets you set the maximum records to return from 1 to 1200 inclusive. The default of 10.
+
+Because the API returns up to 200 records per request, values of limit greater than 200 will result in multiple API calls.
```
Comment (frankfarach) [inline vignettes/npi.Rmd:44]: Suggest revising to indicate that multiple API calls may occur. It depends on how many records are returned by the API. A user may ask for 300 records, but if there are < 200 records returned in the initial GET request, no further request is needed. 

Importantly for the user, the package handles this logic for the user and makes multiple calls as needed.

File: vignettes/npi.Rmd [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+```{r}
+nyc_25 <- npi_search(city = "New York City", limit = 25)
```
Comment (frankfarach) [inline vignettes/npi.Rmd:47]: Can we run this with a limit value greater than 200 to illustrate the auto-paging feature, or will that create problems in a vignette?

**Review by frankfarach — CHANGES_REQUESTED** [review]
All R-CMD-Check jobs now pass, thanks.

Would you mind making the changes mentioned in my review comments? Once addressed I will be happy to merge your PR.

File: vignettes/npi.Rmd [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+This vignette provides an brief introduction to the npi package. 
+
+`npi` is an R package that allows R users to access the [U.S. National Provider Identifier (NPI) Registry](https://npiregistry.cms.hhs.gov/) API by the Center for Medicare and Medicaid Services (CMS). 
+
+The package makes it easy to obtain administrative data linked to a specific individual or organisational healthcare provider. Additionally, users can perform advanced searches based on provider name, location, type of service, credentials, and many other attributes.
+
+## Search registry
+
+To explore organizational providers with primary locations in New York City, we could use the `city` argument in the `npi_search()`. The nyc dataset here finds 10 organizational providers with primary locations in New York City, since 10 is the default number of records that are return in `npi_search()`. The response is a tibble that has high-cardinality data organized into list columns.
```
[the remark this answers is not in the document]
Comment (parmsam) [reply vignettes/npi.Rmd:29]: Done.

File: vignettes/npi.Rmd [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+```{r}
+nyc <- npi_search(city = "New York City")
+nyc
+```
+
+Other search arguments for the function include npi number, enumeration type, taxonomy description, individual provider's first name, individual provider's last name, use first name alias, organization name, address purpose, state, postal code, country code, and limit.
```
[the remark this answers is not in the document]
Comment (parmsam) [reply vignettes/npi.Rmd:36]: Done.

File: vignettes/npi.Rmd [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+```{r setup}
+library(npi)
+``` 
+
+This vignette provides an brief introduction to the npi package. 
+
+`npi` is an R package that allows R users to access the [U.S. National Provider Identifier (NPI) Registry](https://npiregistry.cms.hhs.gov/) API by the Center for Medicare and Medicaid Services (CMS). 
+
+The package makes it easy to obtain administrative data linked to a specific individual or organisational healthcare provider. Additionally, users can perform advanced searches based on provider name, location, type of service, credentials, and many other attributes.
```
[the remark this answers is not in the document]
Comment (parmsam) [reply vignettes/npi.Rmd:25]: Done.

File: vignettes/npi.Rmd (hunk 2 above)
Comment (parmsam) [reply vignettes/npi.Rmd:47]: Looks like we can without any problems. The print is automatically shortened. Done.

**Review by frankfarach — APPROVED** [review]
Thanks for making these changes.

**Pull request merged**